extensions/librdkafka/PublishKafka.cpp

```diff
@@ -266,8 +266,8 @@ class ReadCallback {
     status_ = 0;
     called_ = true;
 
-    gsl_Expects(max_seg_size_ != 0 || (flow_size_ == 0 && "max_seg_size_ == 0 implies flow_size_ == 0"));
-    // ^^ therefore checking max_seg_size_ == 0 handles both division by zero and flow_size_ == 0 cases
+    gsl_Expects(max_seg_size_ != 0 || (flow_size_ == 0 && "max_message_size_ == 0 implies flow_size_ == 0"));
+    // ^^ therefore checking max_message_size_ == 0 handles both division by zero and flow_size_ == 0 cases
```

Comment:
was this renamed by accident? the variable is still `max_seg_size_` so the comments seem to be out of sync now

Reply:
I think it was. I used CLion's rename feature and it affected this file by accident. Restoring.